addons/account_edi_ubl_cii/models/res_config_settings.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, fields
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    ubl_cii_is_generated = fields.Boolean(string='Peppol format', related='company_id.ubl_cii_is_generated', readonly=False)
```

Comment:
Maybe we should give more detail to the user regarding what this action is? Like `Download Peppol format` maybe?